### ISSUE TITLE
Indent rule weirdness

### ISSUE THREAD

**Issue by simov** [eslint/eslint#3547]
```js
{
  'env': {
    'node': true
  },
  'ecmaFeatures': {
    'generators': true
  },
  'rules': {
    // Validate Indentation
    'indent': [2, 2],

    // Enforce Quote Style
    'quotes': [2, 'single'],

    // Enforce Semicolons
    'semi': [2, 'never'],

    // Strict Mode
    'strict': [2, 'global']
  }
}
```

```js
'use strict'

var koa = require('koa')
  , route = require('koa-route')
  , thunkify = require('thunkify')

var qs = require('qs')
```

```
7:1  error  Expected indentation of 2 space characters but found 0  indent
```

How am I supposed to overcome this? Btw just updated to the latest version of eslint 1.2.1

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by simov** [comment]
```js
describe('config', function () {
  describe('credentials', function () {
    it('key secret', function () {
      var provider = {oauth:2, key:'key1', secret:'secret1'}
        , options = {key:'key2', secret:'secret2'}
      config.credentials(provider, options)
      provider.key.should.equal('key1')
      provider.secret.should.equal('secret1')
    })
  })
})
```

```
14:7  error  Expected indentation of 8 space characters but found 6   indent
```

That's the line after the `, options` line. Same config.

**Comment by BYK** [comment]
This is fixed by #3515 but not released yet. Verified that this doesn't happen on `master`.

**Comment by simov** [comment]
Thanks! didn't know about that :+1:
